stables.js

```diff
@@ -32,4 +32,12 @@ console.log(false+true+"NaN");
 console.log("The stable monthly fee is...  " + STABLE_MONTHLY_FEE);
 
 console.log(`My horse's name is "${horseName}" and costs ${STABLE_MONTHLY_FEE} to board him`); // if you use backtick
-console.log("My horse's name is " + horseName + " and he is " + horseAge + " years old now.");
\ No newline at end of file
+console.log("My horse's name is " + horseName + " and he is " + horseAge + " years old now.");
+
+
+
+// LESSON-4 Lab //
```

Comment:
Looks good!! (You cooked w/this girl 😍)
![29s5ao](https://github.com/user-attachments/assets/3a591948-c62c-4e0d-a4db-0b58ad2f892a)